### DISCUSSION TITLE
extracting all conserved sequences from multiple timeseres

### DISCUSSION THREAD

**Discussion by Mschaid** [TDAmeritrade/stumpy#980] [General]
I am looking for methods for extracting all conserved sequences from multiple timer series. As an example, if I have 10 timeseries, and each timeseries has 5 conserved motifs, is there a way to extract all 50 motifs instead of just the 5 conserved sequences? Has any one implemented this? Or put another way, is there a way to get all of the arrays from the clusters produces by the ostinato algorithm?

Thanks,

**Comment by seanlaw** [comment]
@Mschaid Off the top of my head, the only way to accomplish this would be to:

1. Run `ostinato` once on the set of time series
2. Remove/delete the consensus motif from each time series (currently, there is [a bug](https://github.com/TDAmeritrade/stumpy/issues/982) where you cannot simply set those areas within each time series to `np.nan`) and then run `ostinato` again on the modified time series

One challenge of doing it this way is needing to manage indices with changing time series. Until the aforementioned bug gets fixed, I believe that this might be your only option
